src/coreclr/src/interop/inc/interoplib.h

```diff
@@ -45,6 +45,12 @@ namespace InteropLib
         // Reactivate the supplied wrapper.
         HRESULT ReactivateWrapper(_In_ IUnknown* wrapper, _In_ InteropLib::OBJECTHANDLE handle) noexcept;
 
+        // Get the object for the supplied wrapper
+        HRESULT GetObjectForWrapper(_In_ IUnknown *wrapper, _Out_ OBJECTHANDLE *object) noexcept;
```

Comment:
In this code base, let's put the indirection near the type (i.e. `IUnknown *wrapper` -> `IUnknown* wrapper`).

Reply:
~~`_Out_`~~ -> `_Out_result_maybenull_`